images/openstack/Dockerfile.ci

```diff
@@ -11,6 +11,13 @@ COPY --from=builder /go/src/github.com/openshift/installer/images/openstack/rdo-
 COPY --from=builder /go/src/github.com/openshift/installer/images/openstack/rdo-stein.gpg /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Cloud
 COPY --from=registry.svc.ci.openshift.org/origin/4.1:cli /usr/bin/oc /bin/oc
 
+# Install Dependendencies for tests
+# https://github.com/openshift/origin/blob/master/images/tests/Dockerfile.rhel
+RUN yum install --setopt=tsflags=nodocs -y git gzip util-linux && yum clean all && rm -rf /var/cache/yum/* && \
+    git config --system user.name test && \
+    git config --system user.email test@test.com && \
```

Comment:
Mind using `example.com` as a test domain? The one you've used is an actual domain owned by an actual company selling stuff on their website

Reply:
This is taken from https://github.com/openshift/origin/blob/master/images/tests/Dockerfile.rhel#L11-L12

Reply:
(please disregard if pushing again resets the review dance)

Reply:
I didn't want to differ from the original code that Mike linked to.